docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+    cluster_type                 = "REPLICASET"
+
+    # Provider Settings "block"
+    provider_instance_size_name = "M10" # 1 Replication Spec Configuration
+    provider_name               = "AWS" # 2 Provider Settings
+    
+    auto_scaling_disk_gb_enabled = true # 3 Auto Scaling
+    cloud_backup                 = true # 4 Backup Configuration
+
+    replication_specs {
+     num_shards = 1
+     regions_config {
+       region_name     = "US_EAST_1"
+       priority        = 7
+       electable_nodes = 3 # 1 Replication Spec Configuration
+       analytics_nodes = 1 # 1 Replication Spec Configuration
+       read_only_nodes = 0 # 1 Replication Spec Configuration
+       }
+     }
+
+    }
+```
+
+### Example 2: New Configuration (`mongodbatlas_advanced_cluster`)
+
+```terraform
+    resource "mongodbatlas_advanced_cluster" "this" {
+    project_id             = var.project_id
+    name                   = "advanced-cluster"
+    cluster_type           = "REPLICASET"
+    backup_enabled         = true # 4 Backup Configuration
+
+    # Replication specs
+    replication_specs {
+      auto_scaling { # 3 Auto Scaling
+        disk_gb_enabled = true
+      }
+      region_configs {
+        region_name   = "US_EAST_1"
+        priority    = 7
+        provider_name = "AWS" # 2 Provider Settings
+        
+        electable_specs { # 1 Replication Spec Configuration
+            instance_size = "M10"
+            node_count    = 3
+        }
+        analytics_specs { # 1 Replication Spec Configuration
+            instance_size = "M10"
+            node_count    = 1
+        }
+      }
+    }
+```
+
+### Output Changes
+
+- container_id:
+  - before: `mongodbatlas_cluster.this.replication_specs[0].container_id` was a flat string, e.g., `669644ae01bf814e3d25b963`
+  - after: `mongodbatlas_advanced_cluster.this.replication_specs[0].container_id` is a map, e.g., `{"AWS:US_EAST_1": "669644ae01bf814e3d25b963"}`
+
+## How to Change
+Before doing any migration it is recommended to make a backup of your Terraform state files. (ADD_LINK_HERE)
+
+### Method 1: `terraform plan -generate-config-out=cluster.tf`
+This method uses only Terraform native tools and are ideal for customers who:
+1. Have an existing cluster without any Terraform configuration and want to manage their cluster with Terraform.
+2. Have existing `mongodbatlas_cluster` resource(s) and don't want to use an external script for migrating.
+
+#### Steps
+
+1. Find the import IDs of the clusters you want to migrate: `{PROJECT_ID}-{CLUSTER_NAME}`, e.g., `664619d870c247237f4b86a6-legacy-cluster`
+2. Add an import block per cluster to one of your `.tf` files:
+  ```terraform
+  import {
+    to = mongodbatlas_advanced_cluster.this
+    id = "664619d870c247237f4b86a6-legacy-cluster" # from step 1
+  }
+  ```
+3. Run `terraform plan -generate-config-out=cluster.tf`, should generate a `cluster.tf` file and display a message similar to `Plan: 1 to import, 0 to add, 0 to change, 0 to destroy`:
```

Comment:
```suggestion
3. Run `terraform plan -generate-config-out=cluster.tf`. This should generate a `cluster.tf` file and display a message similar to `Plan: 1 to import, 0 to add, 0 to change, 0 to destroy`:
```

Reply:
Thanks updated in 288f7f76a58ef30cf833e0adbb1e9543b719d138